### PR TITLE
add vulnerability-alerts endpoint

### PR DESCRIPTION
**Pull request by cpanato**
Adding vulnerability-alerts endpoint

fixes: https://github.com/google/go-github/issues/1164

### CODE REVIEW

File: github/repos.go [hunk 1]
```diff
@@ -462,6 +462,40 @@ type ListContributorsOptions struct {
 	ListOptions
 }
 
+// EnableVulnerabilityAlerts Enables vulnerability alerts and the dependency graph for a repository.
```
Comment (gmlewis) [inline github/repos.go]: nit: s/Enables/enables/
so that the auto-generated Godocs read nicely.

File: github/repos.go [hunk 2]
```diff
@@ -462,6 +462,40 @@ type ListContributorsOptions struct {
...
+		return nil, err
+	}
+
+	// TODO: remove custom Accept header when this API fully launches
+	req.Header.Set("Accept", mediaTypeRequiredVulnerabilityAlertsPreview)
+
+	return s.client.Do(ctx, req, nil)
+}
+
+// DisableVulnerabilityAlerts Disables vulnerability alerts and the dependency graph for a repository.
```
Comment (gmlewis) [inline github/repos.go]: s/Disables/disables/

File: github/repos.go [hunk 3]
```diff
@@ -462,6 +462,40 @@ type ListContributorsOptions struct {
...
+//
+// GitHub API docs: https://developer.github.com/v3/repos/#disable-vulnerability-alerts
+func (s *RepositoriesService) DisableVulnerabilityAlerts(ctx context.Context, owner string, repository string) (*Response, error) {
```
Comment (gmlewis) [inline github/repos.go]: `owner, repository string`
instead of
`owner string, repository string`

File: github/repos.go [hunk 4]
```diff
@@ -462,6 +462,40 @@ type ListContributorsOptions struct {
 	ListOptions
 }
 
+// EnableVulnerabilityAlerts Enables vulnerability alerts and the dependency graph for a repository.
+//
+// GitHub API docs: https://developer.github.com/v3/repos/#enable-vulnerability-alerts
+func (s *RepositoriesService) EnableVulnerabilityAlerts(ctx context.Context, owner string, repository string) (*Response, error) {
```
Comment (gmlewis) [inline github/repos.go]: `owner, repository string`
instead of
`owner string, repository string`

File: github/repos_test.go [hunk 5]
```diff
@@ -367,6 +367,36 @@ func TestRepositoriesService_Edit_invalidOwner(t *testing.T) {
 	testURLParseError(t, err)
 }
 
+func TestRepositoriesService_EnableVulnerabilityAlerts(t *testing.T) {
+	client, mux, _, teardown := setup()
+	defer teardown()
+
+	mux.HandleFunc("/repos/o/r/vulnerability-alerts", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "PUT")
```
Comment (gmlewis) [inline github/repos_test.go]: In this test and the test below, please also add `testHeader` to check the custom media header (see examples of this throughout the repo).

File: github/repos.go (hunk 2 above)
Comment (cpanato) [reply github/repos.go]: done

File: github/repos.go (hunk 3 above)
Comment (cpanato) [reply github/repos.go]: done

File: github/repos.go (hunk 1 above)
Comment (cpanato) [reply github/repos.go]: done

**Comment by gmlewis** [comment]
Thank you, @gauntface!
Merging.

**Pull request merged**